### ISSUE TITLE
Add "Revanced Extended Beta" Provider

### ISSUE THREAD

**Issue by Felitendo** [anfreire/updateMe-Mobile#34]
Hey it's me again.
On the Revanced Extended GitHub Page are beta versions for example YouTube and YT Music and it would be cool if this Provider could be added to fetch all the beta releases :)

**Comment by anfreire** [comment]
Yes, I'll definitely look into it and add it as soon as possible.

This sounds like a valuable addition that could benefit many users, myself included.
